dataprofiler/profilers/utils.py

```diff
@@ -366,10 +366,10 @@ def biased_kurt(df_series: Series) -> float:
     M4 = 0 if np.abs(M4) < 1e-14 else M4
 
     if M2 == 0:
-        return -3.0
+        return np.float64(-3.0)
 
     with np.errstate(all="ignore"):
-        kurt: float = n * M4 / np.power(M2, 2) - 3
+        kurt: np.float64 = n * M4 / np.power(M2, 2) - 3
```

Comment:
required for `assert_profile_equal` in testing ... otherwise could be a float64 & float